plugin-build/build.gradle.kts

```
@@ -148,6 +154,10 @@ gradlePlugin {
             id = "io.sentry.android.gradle"
             implementationClass = "io.sentry.android.gradle.SentryPlugin"
         }
+        register("kotlinCompilerPlugin") {
+            id = "io.sentry.sentry-kotlin-compiler-gradle-plugin"
+            implementationClass = "io.sentry.SentryKotlinCompilerGradlePlugin"
```

```suggestion
            implementationClass = "io.sentry.kotlin.gradle.SentryKotlinCompilerGradlePlugin"
```
maybe worth to change package to adhere to the android gradle plugin structure?